docs/how_to/list_available_backends.ipynb

```diff
@@ -0,0 +1,99 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# List available backends\n",
+    "\n",
+    "This guide shows you how to list all the backends available to you.\n",
+    "\n",
+    "## Before you begin\n",
+    "\n",
+    "Throughout this guide, we will assume that you have [setup the Qiskit Runtime service instance](../getting_started.rst)  and initialize it as `service`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "service = QiskitRuntimeService()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Steps\n",
+    "\n",
+    "You can see the list of available backends by calling ``QiskitRuntimeService.backends()`` :"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<IBMBackend('ibmq_qasm_simulator')>,\n",
+       " <IBMBackend('simulator_stabilizer')>,\n",
+       " <IBMBackend('simulator_mps')>,\n",
+       " <IBMBackend('simulator_extended_stabilizer')>,\n",
+       " <IBMBackend('simulator_statevector')>]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "service.backends()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "\n",
+    "The output will depends on the channel (IBM Quantum or IBM Cloud) you are accessing Qiskit Runtime, the provider (for IBM Quantum) or the plan (for IBM Cloud).\n",
```

Comment:
We don't refer to hub/group/project as provider anymore in IBM Quantum. It could be called plan as well but until then, I would just refer to it as hub/group/project.

Reply:
```suggestion
    "The output will depends on the channel (IBM Quantum or IBM Cloud) you are accessing Qiskit Runtime, the hub/group/project (for IBM Quantum) or the plan (for IBM Cloud).\n",
```